src/osd/OSD.cc

```diff
@@ -6808,7 +6808,12 @@ OSDService::ScrubJob::ScrubJob(CephContext* cct,
     double r = rand() / (double)RAND_MAX;
     sched_time +=
       scrub_min_interval * cct->_conf->osd_scrub_interval_randomize_ratio * r;
-    deadline += scrub_max_interval;
+    if (scrub_max_interval < 0) {
```

Comment:
can we change  make this condition `(scrub_max_interval == 0)` ? i think, in future, the `interval`s in `options.cc` will be `ceph::timespan`s. so if we change this way, it'd be easier to do this in future.

Reply:
ok